src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,12 +32,28 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
+
   // The robot's subsystems are defined here...
   private final Drivetrain m_drive = new Drivetrain();
   private final FourBarArm m_arm = new FourBarArm();
 
   // Shuffleboard stuff
   SendableChooser<Command> m_autoCommand = new SendableChooser<>();
+  SendableChooser<Alliance> m_allianceChooser = new SendableChooser<>();
```

Comment:
I might have already said this, I'm not sure where the comment went, but just use Driverstation.getAlliance()